code/modules/mob/living/simple_animal/abnormality/!nonstandard/scp/tickle_monster.dm

```diff
@@ -0,0 +1,83 @@
+/mob/living/simple_animal/hostile/abnormality/tickle_monster
+	name = "The Tickle Monster"
+	desc = "An orange blob that is always looking for a hug."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "999"
+	icon_living = "999"
+	is_flying_animal = TRUE
+	maxHealth = 600	//It is helpful and therefore weak.
+	health = 600
+	//work stuff
+	can_breach = TRUE
+	start_qliphoth = 4
+	threat_level = ZAYIN_LEVEL
+	work_damage_amount = 4
+	work_damage_type = WHITE_DAMAGE
+
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 2)
+	melee_damage_lower = 0
+	melee_damage_upper = 0
+	melee_damage_type = WHITE_DAMAGE
```

Comment:
If you wanted, you could have made this a negative number and made it lose target of someone after hitting them enough times. The negative number would actually heal sanity. Not something that needs changing just a comment.